src/debug/clean-pending-payments-for-dead-node.ts

```diff
@@ -33,7 +33,8 @@ const listAllPendingPayments = async (): Promise<
 
     return results.filter((tx) => tx.debit > 0).map((tx) => translateToLedgerTx(tx))
   } catch (err) {
-    return new UnknownLedgerError(err)
+    if (err instanceof Error) return new UnknownLedgerError(err.message)
+    return new UnknownLedgerError()
```

Comment:
Why not pass the err in either way? We need to be alerted / find out about the details of the error and not eat it.

Reply:
If you open this up in vscode you get a type error if you try to pass in err like `return new UnknownLedgerError(err)`. `err` is type `unknown` and `UnknownLedgerError` is expecting a `string | undefined`. One of my original ideas was to check if all errors where type `string` but i think @nicolasburtey had doubts about it. Maybe we could check like this:

```
function isString(value: unknown): value is string {
  return typeof value === "string"
}
```

and use like this:
```
return new UnknownLedgerError(isString(err) ? err : undefined)
```